docs/release.md

```diff
@@ -42,7 +42,8 @@ a 1.0.0 release.
 * None.
 
 ##### Enhancements
-* None.
+* Updated PowerTransformer proto to include powerTransformerInfo property.
```

Comment:
you didn't actually do this, so i assume the intention is to use the assetInfo field on PowerSystemResource?
if so you can leave out this line i think.

Reply:
He had done this and I told him he didn't need it because of the assetInfo field on PSR. Obviously forgot to revert this.

Reply:
Yep, it is a leftover, I'll remove the release note.